js/main.js

```diff
@@ -0,0 +1,66 @@
+// We create an empty array to store title and author
+let books = [];
+// select an element with the required class
+const bookList = document.querySelector('.book-List');
+const addForm = document.querySelector('.book-form');
+
+const createBookElement = (book) => {
+  // create elements
+  const bookContainer = document.createElement('div');
+  const titleElement = document.createElement('p');
+  const authorElement = document.createElement('p');
+  const removeBtn = document.createElement('button');
+  // create text nodes
+  const title = document.createTextNode(book.title);
+  const author = document.createTextNode(book.author);
+  const remove = document.createTextNode('Remove');
+  // add nodes to created elements
+  titleElement.appendChild(title);
+  authorElement.appendChild(author);
+  removeBtn.appendChild(remove);
+  // add both p and button to bookContainer(div)
+  bookContainer.appendChild(titleElement);
+  bookContainer.appendChild(authorElement);
+  bookContainer.appendChild(removeBtn);
+
+  // Create function in remove buton
+  removeBtn.addEventListener('click', () => {
+    books = books.filter((currentBook) => book.title !== currentBook.title);
+    localStorage.setItem('books', JSON.stringify(books));
+    bookContainer.remove();
+  });
```

Comment:
[OPTIONAL]
- [ ] I suggest that you use a unique `id` property for each book, it would make it easier to remove books with duplicate values.